app/models/grda_warehouse/synthetic/assessment.rb

```diff
@@ -80,14 +80,22 @@ def hud_assessment_hash
         AssessmentType: assessment_type,
         AssessmentLevel: assessment_level,
         PrioritizationStatus: prioritization_status,
-        DateCreated: source.created_at,
-        DateUpdated: source.updated_at,
+        DateCreated: created_at_from(source),
+        DateUpdated: updated_at_from(source),
         UserID: user_id,
         data_source_id: enrollment.data_source_id,
         synthetic: true,
       }
     end
 
+    private def created_at_from(source)
```

Comment:
suggest adding a clarifying comment
```suggestion
    # overridden in subclass, SyntheticCeAssessment::EnrollmentCeAssessment
    private def created_at_from(source)
```